### PR TITLE
#166179906 Added e-mail notification to debit and credit routes

### PR DESCRIPTION
**Pull request by encodedBicoding**
#### What does this PR do?
It creates e-mail notification when a user account is debited or credited.

#### Description of Task to be completed?
1. Used `notify` function to create debit and credit mail notification

#### What are the relevant pivotal tracker stories?
[Link to PT Story](https://www.pivotaltracker.com/story/show/166179906)

### CODE REVIEW

File: server/controllers/userController.js [hunk 1]
```diff
@@ -126,6 +126,7 @@ export default class UsersController {
 
   static async signin(req, res) {
     const { email, password } = req.body;
+    console.log(req.body)
```
Comment (OvieMudi) [inline server/controllers/userController.js]: Remove console statement

Comment (encodedBicoding) [reply server/controllers/userController.js]: done

**Review by OvieMudi — COMMENTED** [review]
Add .idea folder to gitinore
Remove console log statement

File: server/controllers/userController.js (hunk 1 above)
Comment (OvieMudi) [reply server/controllers/userController.js]: Cool. Expecting your push

**Review by OvieMudi — APPROVED** [review]

**Pull request merged**